server/subsonic/stream.go

```diff
@@ -97,6 +97,27 @@ func (c *StreamController) Download(w http.ResponseWriter, r *http.Request) (*re
 		return nil, err
 	}
 
+	maxBitRate := utils.ParamInt(r, "bitrate", 0)
+	format := utils.ParamString(r, "format")
+
+	if format == "" {
+		if conf.Server.AutoTranscodeDownlaod {
```

Comment:
"Download" also spelled wrong here.
```suggestion
		if conf.Server.AutoTranscodeDownload {
```